frigate/track/norfair_tracker.py

```diff
@@ -273,9 +273,11 @@ def match_and_update(self, frame_time, detections):
                 min(self.detect_config.width - 1, estimate[2]),
                 min(self.detect_config.height - 1, estimate[3]),
             )
+            estimate_velocity = tuple(t.estimate_velocity.flatten().astype(int))
```

Comment:
should we only calculate this if this camera has autotracking enabled?

Reply:
This is just pulling an existing field already present on every Norfair detection. The rest of the frigate codebase doesn't use it, so it shouldn't cause any problems just to have in the `obj` object.

Reply:
Oh okay cool, wasn't sure if retrieving that value started the calcuation

Reply:
For reference: https://tryolabs.github.io/norfair/2.2/reference/tracker/#norfair.tracker.TrackedObject.estimate_velocity